tools/provision.sh

```diff
@@ -17,18 +17,10 @@ sudo cp -a /usr/share/tests/osbuild-composer/composer/*.toml \
 
 # Copy rpmrepo snapshots for use in weldr tests
 sudo mkdir -p /etc/osbuild-composer/repositories
-# Copy all fedora repo overrides
-sudo cp -a /usr/share/tests/osbuild-composer/repositories/{fedora,centos}-*.json \
-    /etc/osbuild-composer/repositories/
-# RHEL nightly repos need to be overriden in rhel-8.json and rhel-8-beta.json
-case "${ID}-${VERSION_ID}" in
-    "rhel-8.4")
-        # Override old rhel-8.json and rhel-8-beta.json because RHEL 8.4 test needs nightly repos
-        sudo cp /usr/share/tests/osbuild-composer/repositories/rhel-84.json /etc/osbuild-composer/repositories/rhel-8.json
-        # If multiple tests are run and call provision.sh the symlink will need to be overriden with -f
-        sudo ln -sf /etc/osbuild-composer/repositories/rhel-8.json /etc/osbuild-composer/repositories/rhel-8-beta.json;;
-    *) ;;
-esac
+# This scirpt is now RHEL-8.4 specific, install the only override:
+sudo cp /usr/share/tests/osbuild-composer/repositories/rhel-84.json /etc/osbuild-composer/repositories/rhel-8.json
```

Comment:
This will then be pointing to latest 8.4 development compose. I think you want to use cdn repos, so you want this here https://github.com/osbuild/osbuild-composer/commit/b5987a5ca51826f29a3bce742d693a55f16f016f

Reply:
Actually I'd prefer to keep rpmrepo because the regression test for the satellite issue creates an http proxy that forwards the requests to the repository configured in this file and it doesn't have support for client-side SSL certificates which is required for the CDN repository.